README.md

```diff
@@ -48,6 +48,27 @@ Snapshotting configuration; setting values in this list will save the database t
 
 Database compression and location configuration.
 
+    redis_maxmemory: 0
+    
+Limit memory usage to the specified amount of bytes. When the memory limit is reached Redis will try to remove keys according to the eviction policy selected (see redis_maxmemory_policy).
+Leave at 0 for no limit.
+
+    redis_maxmemory_policy: "allkeys-lru"
```

Comment:
Is this the default shipped with Redis? (I just want to make sure we don't change people's settings if they're already using the defaults).